packages/typography/src/elements/CodeBlock.tsx

```diff
@@ -24,9 +24,45 @@ interface IToken {
   empty?: boolean;
 }
 
+/* until https://github.com/FormidableLabs/prism-react-renderer/pull/127 is available */
+const LANGUAGES = [
+  'markup',
+  'bash',
+  'clike',
+  'c',
+  'cpp',
+  'css',
+  'javascript',
+  'jsx',
+  'coffeescript',
+  'actionscript',
+  'css-extr',
+  'diff',
+  'git',
+  'go',
+  'graphql',
+  'handlebars',
+  'json',
+  'less',
+  'makefile',
+  'markdown',
+  'objectivec',
+  'ocaml',
+  'python',
+  'reason',
+  'sass',
+  'scss',
+  'sql',
+  'stylus',
+  'tsx',
+  'typescript',
+  'wasm',
+  'yaml'
+] as const;
+
 export interface ICodeBlockProps extends HTMLAttributes<HTMLPreElement> {
   /** Selects the language used by the [Prism](https://prismjs.com/) tokenizer */
-  language?: Language;
+  language?: typeof LANGUAGES[number];
```

Comment:
Along with aligning to the specified array, this construct has the affect of improving API output for https://garden.zendesk.com/components/code-block#codeblock.